manifests/client/install.pp

```diff
@@ -5,7 +5,19 @@
     fail 'class ::openldap::client has not been evaluated'
   }
 
-  package { $::openldap::client::package:
-    ensure => present,
+  case $::osfamily {
+    'Debian': {
+      package { $::openldap::client::package:
+        ensure => present,
+      }
+    }
+    'RedHat': {
+      class { '::openldap::client::utils':
```

Comment:
We may have to use `include ::openldap::client::utils` here to avoid duplicate declaration on `Class[openldap::client::utils]`